edx_solutions_api_integration/users/serializers.py

```diff
@@ -206,3 +207,20 @@ def get_course(self, enrollment):
             ), None
         )
         return course_overview
+
+
+class ExtendedOrganizationSerializer(BasicOrganizationSerializer):
+    """ Serializer for Organization with mobile apps and themes data """
+    mobile_apps = MobileAppSerializer(many=True)
+    themes = serializers.SerializerMethodField()
+
+    class Meta(BasicOrganizationSerializer.Meta):
```

Comment:
extending Meta class from `BasicOrganizationSerializer.Meta` does not make sense here. what if we define this meta class as `class Meta:`?